packages/core/test/crypto.spec.ts

```diff
@@ -136,98 +132,51 @@ describe("Crypto", () => {
-      const invalidCrypto = new Crypto(core, logger);
-      expect(() => invalidCrypto.encrypt("topic", "message")).to.throw("Not initialized. crypto");
-    });
-    it("throws if the passed topic is not known", () => {
-      const topic = utils.generateRandomBytes32();
-      expect(() => crypto.encrypt(topic, "message")).to.throw();
-    });
-    it("resolves symKey from `topic` param and encrypts with `message` param", async () => {
-      const message = "some message";
-      const symKey = utils.generateRandomBytes32();
-      // Set a topic-symKey pair in the keychain to later retrieve via `encrypt`.
-      const topic = await crypto.setSymKey(symKey);
-      const spy = Sinon.spy();
-      // @ts-ignore
-      utils.encrypt = spy;
-      crypto.encrypt(topic, message);
-      const [payload] = spy.getCall(0).args;
-      expect(payload).to.deep.equal({ symKey, message });
-    });
-  });
-
-  describe("decrypt", () => {
-    it("throws if not initialized", () => {
-      const invalidCrypto = new Crypto(core, logger);
-      expect(() => invalidCrypto.decrypt("topic", "encoded")).to.throw("Not initialized. crypto");
-    });
-    it("throws if the passed topic is not known", () => {
-      const topic = utils.generateRandomBytes32();
-      expect(() => crypto.decrypt(topic, "encoded")).to.throw();
-    });
-    it("resolves symKey from `topic` param and decrypts `encoded` param", async () => {
-      const encoded = "encoded";
-      const symKey = utils.generateRandomBytes32();
-      // Set a topic-symKey pair in the keychain to later retrieve via `decrypt`.
-      const topic = await crypto.setSymKey(symKey);
-      const spy = Sinon.spy();
-      // @ts-ignore
-      utils.decrypt = spy;
-      crypto.decrypt(topic, encoded);
-      const [payload] = spy.getCall(0).args;
-      expect(payload).to.deep.equal({ symKey, encoded });
-    });
-  });
-
   describe("encode", () => {
+    const symKey = "5720435e682cd03ee45b484f9a213f0e3246a0ccc2cca183b72ab1cbfbefb702";
     const payload = { id: 1, jsonrpc: "2.0", result: "result" };
+    // const encoded =
+    //   "AG7iJl9mMl9K04REnuWaKLQU6kwMcQWUd69OxGOJ5/A+VRRKkxnKhBeIAl4JRaIft3qZKEfnBvc7/Fife1DWcERqAfJwzPI=";
 
-    it("throws if not initialized", () => {
+    it("throws if not initialized", async () => {
       const invalidCrypto = new Crypto(core, logger);
-      expect(() => invalidCrypto.encode("topic", payload)).to.throw("Not initialized. crypto");
-    });
-    it("encodes `payload` as hex string if the passed topic is not known", () => {
-      const topic = utils.generateRandomBytes32();
-      const result = crypto.encode(topic, payload);
-      expect(result).to.equal(encoding.utf8ToHex(safeJsonStringify(payload)));
+      await expect(invalidCrypto.encode("topic", payload)).to.eventually.be.rejectedWith(
+        "Not initialized. crypto",
+      );
     });
     it("encrypts `payload` if the passed topic is known", async () => {
```

Comment:
```suggestion
    it.skip("encrypts `payload` if the passed topic is known", async () => {
```

Probably better to skip here for now if we don't have any `expect` assertions to avoid a false positive pass.